### PR TITLE
Remove Niassa plugin

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

**Review by avarsava — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 * Adds metrics for cache refresh start and end times
 
+Niassa plugin:
+
+* Obliterate this plugin and any references to that nightmare.
```
Comment (callunity) [inline RELEASE_NOTES.md:7]: While I fully understand where this is coming from, can the language please be modified to something more neutral? `Obliterate this plugin and any references to that nightmare` -> `Obliterate this plugin and any references to it`

Comment (apmasell) [reply RELEASE_NOTES.md:7]: Patches welcome.

File: demo/misc.constants [hunk 2]
```diff
@@ -1,8 +1,6 @@
 {
-  "cromwell_host": "http://cromwell.example.com:8000",
```
Comment (callunity) [inline demo/misc.constants:2]: Why remove the `cromwell_host` argument here?

File: faq.md [hunk 3]
```diff
@@ -28,18 +28,18 @@ From _Tools_, select _Type Converter_ and enter the type descriptor in the box
...
 
 To get Shesmu to replace the actions, purge the old ones (either individually or in bulk). This will cause the new action to be in the scheduler.
 
-For Niassa workflows, it uses the workflow SWID, the LIMS keys, and any annotations. If your olive only changes INI parameters, the modified actions will not displace the old actions. Even if purged, the new action will still match the existing Niassa workflow runs. If those need to be re-run, click the _Skip and Re-run_ button on the action.
+For Vidarr workflows, it uses the workflow, the input files, the LIMS keys, and labels. If your olive only changes arguments, the modified actions will not displace the old actions. Even if purged, the new action will still match the existing Vidarr workflow runs. If those need to be re-run, click the _Unload_ button on the action.
```
Comment (callunity) [inline faq.md:42]: There's `Delete and Purge` available for failed actions, but no `Unload` button available for SUCCEEDED actions. Perhaps remove this last sentence until we implement the `Unload` button?

File: demo/misc.constants (hunk 2 above)
Comment (apmasell) [reply demo/misc.constants:2]: It's not longer used in any of the demo olives.

**Review by callunity — APPROVED** [review]

**Pull request merged**